scm/etc/scm_qsub_example.py

```diff
@@ -3,10 +3,15 @@
 -----------------------------------------------------------------------
  Description:  Example script to submit a job through the HPC batch system
 
- Assumptions: For use on Cheyenne. This script must be copied to the bin directory.
+ Assumptions: For use on Derecho. This script must be copied to the bin directory.
```

Comment:
Has this been tested on Derecho? I don't think I ever tried out this example script in my Derecho testing.

Reply:
Generalized this script more, testing right now, thanks for your input!